Makefile

```diff
@@ -15,7 +15,7 @@ vet:
 	go vet ./...
 
 test:
-	go test -parallel=4 ./...
+	go test -parallel=4 -count=1 ./...
```

Comment:
What does this do?

Reply:
from `go help test`: `To disable test caching, use any test flag
or argument other than the cacheable flags. The idiomatic way to disable
test caching explicitly is to use -count=1.`